src/utils/Mock.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+// mock data
+const addressData = require('../offlineData/addressData.json');
+const appHeaderNavigationData = require('../offlineData/appHeaderNavigationData.json');
+const cartData = require('../offlineData/cartData.json');
+const checkoutData = require('../offlineData/checkoutData.json');
+const countryData = require('../offlineData/countryData.json');
+const emailFormData = require('../offlineData/emailFormData.json');
+const orderReviewData = require('../offlineData/orderReviewData.json');
+const purchaseReceiptData = require('../offlineData/purchaseReceiptData.json');
+const searchData = require('../offlineData/searchData.json');
+const searchFormData = require('../offlineData/searchFormData.json');
+const searchProduct1 = require('../offlineData/searchProduct1.json');
+const searchProduct2 = require('../offlineData/searchProduct2.json');
+const womensCategoryItemsData = require('../offlineData/womensCategoryItemsData.json');
+const womensProduct1 = require('../offlineData/womensProduct1.json');
+const womensProduct2 = require('../offlineData/womensProduct2.json');
+const womensProduct3 = require('../offlineData/womensProduct3.json');
+const womensProduct4 = require('../offlineData/womensProduct4.json');
+const womensProduct5 = require('../offlineData/womensProduct5.json');
+const womensProduct6 = require('../offlineData/womensProduct6.json');
+const womensProduct7 = require('../offlineData/womensProduct7.json');
+const womensProduct8 = require('../offlineData/womensProduct8.json');
+
+// followlocation
+// eslint-disable-next-line max-len
+const purchaseReceiptFollow = 'http://localhost:8080/cortex/purchases/orders/vestri/ga3wgmbxgi2wcljxga4taljuha4dcljymyzdcljrgnswgolggnqwkntdge=/form?followlocation&zoom=appliedpromotions:element,billingaddress,discount,lineitems:element,lineitems:element:options:element,lineitems:element:options:element:value,paymentmeans:element,shipments:element:destination,shipments:element:shippingoption';
```

Comment:
I'm a bit hesitant about setting localhost:8080 as the host to hit. In case we ever switch the port in the webpack config, these values here will still be hardcoded. Lets see how everyone else feels though

Reply:
that's a good point, however if the port changes, all of our static data becomes invalid. since they all are recording the ports the request was sent with.

Reply:
Exactly, but we could fix that with a placeholder & regex. I'm just curious though about this and would like to get more opinions on it. @dusanradovanovic @jatinbehlEP what do you think?